assets/queries/ansible/aws/sql_analysis_services_port_2383_is_publicly_accessible/query.rego

```diff
@@ -0,0 +1,35 @@
+
+package Cx
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+    awsEc2 := task["amazon.aws.ec2_group"]
+    rules := awsEc2.rules[j]
+    port := rules.ports[k]
+    rules.cidr_ip == "0.0.0.0/0"
+    portNumber := 2383
+    port <= portNumber
+    port >= portNumber
```

Comment:
here your logic doesn't make sense, this will only pass if var 'port' is equal to var 'portNumber', so why not 
'port == portNumber'